css/src/components/form/form.scss

```diff
@@ -0,0 +1,127 @@
+/* stylelint-disable max-nesting-depth, selector-max-specificity  */
+
+$form-font-size: $font-size-7;
+$form-font-size-sm: $font-size-8;
+$form-font-size-lg: $font-size-6;
+
+$icon-size: 2.25em;
+
+$icon-color: $secondary-background !default;
+$icon-active-color: $secondary !default;
+
+.field {
+	position: relative;
+	font-size: $form-font-size;
+	text-align: start;
+
+	&:not(:last-child) {
+		margin-block-end: 0.75rem;
+	}
+
+	.field-label {
+		margin-block-end: $spacer-3;
+
+		@include tablet {
+			flex: 1 0 0;
+		}
+	}
+
+	.field-body {
+		&.field-body-icons {
```

Comment:
related to the other comment- let's remove field-body-icons. Input addons/icons can be part of the input styles if we need them in the future.

I think the hierarchy here is:

- .field: a `<div>` or a `<fieldset>`, depending on what the .field-body content is.
   - .field-label: a `<label>` or `<legend>`, depending on what the .field-body content is.
   - .field-body
      - anything... an [input](https://design.docs.microsoft.com/components/input.html), [radios](https://design.docs.microsoft.com/components/radio.html), [checkboxes](https://design.docs.microsoft.com/components/checkbox.html)

Reply:
We do have several use cases in docs so we do need some sort of replacement. I moved icons to inputs.

Reply:
Removed icon changes from this PR and created a separate PR https://github.com/microsoft/atlas-design/pull/411